### PR TITLE
Added logic for endpoint

### PR DESCRIPTION
**Pull request by ParT1s**

### CODE REVIEW

File: src/NewsService.Business/GetNewsCommand.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            [FromServices] IMapper<DbNews, News> mapper)
+        {
+            this.repository = repository;
+            this.mapper = mapper;
+        }
+
+        public News Execute(Guid newsId)
+        {
+            var dbNews = repository.GetNews(newsId);
+            return mapper.Map(dbNews);
```
Comment (djino97) [inline src/NewsService.Business/GetNewsCommand.cs:29]: Add empty line

File: src/NewsService.Data/NewsService.Data.csproj [hunk 2]
```diff
@@ -6,6 +6,10 @@
     <RootNamespace>LT.DigitalOffice.NewsService.Data</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.8" />
```
Comment (djino97) [inline src/NewsService.Data/NewsService.Data.csproj:10]: Update kernel

**Comment by KostinP** [comment]
You have not updated the Kernel everywhere, so it is not going to

**Review by KostinP — APPROVED** [review]

File: src/NewsService/Configuration/RabbitMqConfig.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using LT.DigitalOffice.Kernel.Broker;
+
+namespace LT.DigitalOffice.ProjectService.Configuration
+{
+    public class RabbitMqConfig : BaseRabbitMqOptions
```
Comment (KostinP) [inline src/NewsService/Configuration/RabbitMqConfig.cs:5]: Remove this file and specify the base class in the startup

File: src/NewsService.Models.Dto/News.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace LT.DigitalOffice.NewsService.Models.Dto
+{
+    public class News
```
Comment (KostinP) [inline src/NewsService.Models.Dto/News.cs:8]: Use News model from develop

File: src/NewsService.Data/NewsRepository.cs [hunk 5]
```diff
@@ -38,5 +39,16 @@ public Guid CreateNews(DbNews news)
 
             return news.Id;
         }
+
+        public DbNews GetNews(Guid newsId)
+        {
+            var news = provider.News.FirstOrDefault(x => x.Id == newsId);
+            if (news == null)
```
Comment (djino97) [inline src/NewsService.Data/NewsRepository.cs:46]: Add empty line

File: test/NewsService.Mappers.UnitTests/NewsMapperTests.cs [hunk 6]
```diff
@@ -54,31 +62,49 @@ public void OneTimeSetUp()
...
-            SerializerAssert.AreEqual(expectedDbNews, dbNews);
+            SerializerAssert.AreEqual(this.dbNews, dbNews);
         }
         #endregion
+
+        #region DbNews to News
+        [Test]
+        public void ShouldThrowArgumentNullExceptionWhenDbNewsIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() => dbNewsMapper.Map(null));
```
Comment (djino97) [inline test/NewsService.Mappers.UnitTests/NewsMapperTests.cs:96]: move null from arguments and initialize a variable with  null value

**Pull request closed without merging**